### PR TITLE
fix(logs): show addon logs when addonId is provided

### PR DESCRIPTION
**Pull request by miton18**

### CODE REVIEW

File: src/commands/logs.js [hunk 1]
```diff
@@ -5,24 +5,27 @@ const LogV2 = require('../models/log.js');
 const Log = require('../models/log-v4.js');
 const Logger = require('../logger.js');
 const { Deferred } = require('../models/utils.js');
+const colors = require('colors/safe');
 
 async function appLogs (params) {
-  const { alias, after: since, before: until, search, 'deployment-id': deploymentId } = params.options;
+  const { alias, addon, after: since, before: until, search, 'deployment-id': deploymentId } = params.options;
```
Comment (hsablonniere) [inline src/commands/logs.js:11]: suggestion: we could alias it here to `addonId` and reuse it directly on line 22

File: src/commands/logs.js [hunk 2]
```diff
@@ -5,24 +5,27 @@ const LogV2 = require('../models/log.js');
...
   const { appId, ownerId } = await AppConfig.getAppDetails({ alias });
 
-  Logger.println('Waiting for application logs…');
-
   // TODO: drop when addons are migrated to the v4 API
-  if (params.addon) {
+  if (addon) {
+
+  Logger.println(colors.blue('Waiting for addon logs…'));
     const { addon: addonId } = params.options;
```
Comment (hsablonniere) [inline src/commands/logs.js:22]: nitpick: no need to get it again from params.options

**Review by hsablonniere — APPROVED** [review]
Hey @miton18, thanks for this!!

You changed the color in the same commit, that's OK but I almost missed. I added a suggestion but it's not blocking, feel free to apply it or not and merge + release!

File: src/commands/logs.js [hunk 3]
```diff
@@ -5,24 +5,25 @@ const LogV2 = require('../models/log.js');
...
   const { appId, ownerId } = await AppConfig.getAppDetails({ alias });
 
-  Logger.println('Waiting for application logs…');
-
   // TODO: drop when addons are migrated to the v4 API
-  if (params.addon) {
-    const { addon: addonId } = params.options;
+  if (addonId) {
+    Logger.println(colors.blue('Waiting for addon logs…'));
     const appAddonId = addonId || await AppConfig.getAppDetails({ alias }).then(({ appId }) => appId);
```
Comment (hsablonniere) [inline src/commands/logs.js:21]: Do we still need this?

**Review by hsablonniere — CHANGES_REQUESTED** [review]
Last question, sorry.

File: src/commands/logs.js (hunk 3 above)
Comment (miton18) [reply src/commands/logs.js]: Indeed, dropped

**Review by hsablonniere — APPROVED** [review]
LGTM :+1:

**Pull request merged**